import static org.junit.Assert.*;

import org.junit.Before;
import org.junit.Test;


public class ApplicationTest {

	@Before
	public void setUp() throws Exception {
	}

	@Test
	public void testGetStudents() {
		fail("Not yet implemented");
	}

	@Test
	public void testGetCourses() {
		fail("Not yet implemented");
	}

	@Test
	public void testApplication() {
		fail("Not yet implemented");
	}

	@Test
	public void testApplicationListOfStudentListOfCourse() {
		fail("Not yet implemented");
	}

	@Test
	public void testApplicationString() {
		fail("Not yet implemented");
	}

	@Test
	public void testLogIn() {
		fail("Not yet implemented");
	}

	@Test
	public void testLogOut() {
		fail("Not yet implemented");
	}

	@Test
	public void testCreateCourse() {
		fail("Not yet implemented");
	}

	@Test
	public void testCreateUnit() {
		fail("Not yet implemented");
	}

	@Test
	public void testAddUnitToCourse() {
		fail("Not yet implemented");
	}

	@Test
	public void testAddSubSectionToUnit() {
		fail("Not yet implemented");
	}

	@Test
	public void testCreateNote() {
		fail("Not yet implemented");
	}

	@Test
	public void testAddNoteToUnit() {
		fail("Not yet implemented");
	}

	@Test
	public void testCreateExercise() {
		fail("Not yet implemented");
	}

	@Test
	public void testStudents() {
		fail("Not yet implemented");
	}

	@Test
	public void testGetTeacher() {
		fail("Not yet implemented");
	}

	@Test
	public void testSetTeacher() {
		fail("Not yet implemented");
	}

	@Test
	public void testGetCurrentUser() {
		fail("Not yet implemented");
	}

}
